app/pages/sections/catalogue-solutions/catalogue-solutions/controller.test.js

```diff
@@ -27,16 +27,23 @@ describe('catalogue-solutions controller', () => {
       getData.mockResolvedValueOnce({});
 
       await getCatalogueSolutionsPageContext({ orderId, accessToken });
-      expect(getData.mock.calls.length).toEqual(1);
+      expect(getData.mock.calls.length).toEqual(2);
       expect(getData).toHaveBeenCalledWith({
         endpoint: `${orderApiUrl}/api/v1/orders/order-id/sections/catalogue-solutions`,
```

Comment:
can you also change this to call the new endpoint please like with additional-services

Reply:
Well spotted!